client/app/bundles/course/survey/pages/SurveyResults/index.tsx

```diff
@@ -0,0 +1,183 @@
+import { FC, useEffect, useState } from 'react';
+import { defineMessages } from 'react-intl';
+import {
+  Card,
+  CardContent,
+  FormControlLabel,
+  ListSubheader,
+  Switch,
+  Typography,
+} from '@mui/material';
+
+import { fetchResults } from 'course/survey/actions/surveys';
+import surveyTranslations from 'course/survey/translations';
+import CourseUserTypeTabs, {
+  CourseUserType,
+  CourseUserTypeTabValue,
+  getCurrentSelectedUserType,
+} from 'lib/components/core/CourseUserTypeTabs';
+import LoadingIndicator from 'lib/components/core/LoadingIndicator';
+import { useAppDispatch, useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import withSurveyLayout from '../../containers/SurveyLayout';
+
+import ResultsSection from './ResultsSection';
+
+const translations = defineMessages({
+  results: {
+    id: 'course.survey.SurveyResults.results',
+    defaultMessage: 'Results',
+  },
+  includePhantoms: {
+    id: 'course.survey.SurveyResults.includePhantoms',
+    defaultMessage: 'Include Phantom Students',
```

Comment:
`Include phantom users` to align with `course.assessment.submission.SubmissionsIndex.includePhantoms`